cli/src/main/java/keywhiz/cli/Printing.java

```diff
@@ -112,16 +112,40 @@ public void printSanitizedSecretWithDetails(SanitizedSecret secret, List<String>
           .forEach(c -> System.out.println(INDENT + c.getName()));
     }
 
-    if (options.contains("metadata")) {
+    if (options.contains("secret_data")) {
       System.out.println("\tMetadata:");
       if(!secret.metadata().isEmpty()) {
         System.out.println(INDENT + secret.metadata().toString());
```

Comment:
While you're in here, can we change secret.metadata() to be JSON instead of toString?

I think something like this will work:

      if(!secret.metadata().isEmpty()) {
        String metadata;
        try {
          metadata = new ObjectMapper().writeValueAsString(secret.metadata());
        } catch (JsonProcessingException e) {
          throw Throwables.propagate(e);
        }
        System.out.println(INDENT + metadata);
      }

Doesn't have to be in this PR